spec/daemon_spec.rb

```diff
@@ -2,45 +2,99 @@
 describe ServerEngine::Daemon do
   include_context 'test server and worker'
 
-  it 'run and graceful stop' do
-    dm = Daemon.new(TestServer, TestWorker, daemonize: true, pid_path: "tmp/pid")
-    dm.main
+  unless ServerEngine.windows?
```

Comment:
Yeah, I can understand your opinion, but Rspec2 doesn't have any API to claim that "This shouldn't be executed". It was supported at Rspec3 (`skip`).
So, could you use `pending` for now? We can convert Rspec2 test code into Rspec3 style with using transpec, and it can handle `pend`, and can't handle `unless` block.